common.yml

```diff
@@ -2125,9 +2134,9 @@ buildvariants:
     USE_SSL: "true"
   tasks:
   - name: "unit"
-  - name: ".3.4"
-  - name: ".3.6"
-  - name: ".4.0"
+  - name: ".3.4 !.mmap"
+  - name: ".3.6 !.mmap"
+  - name: ".4.0 !.mmap"
```

Comment:
what's the reason for this change?

Reply:
Because when I check the tests like this one
https://evergreen.mongodb.com/task_log_raw/mongo_tools_ubuntu1604_arm64_integration_4.0_mmapv1_patch_5fb51b831d623729dbc2b68ad97575d8464e1903_60ec5b252fbabe24ffe508ad_21_07_12_15_09_51/0?type=T
The server complains 
```
Cannot start server with an unknown storage engine: mmapv1, terminating
```
 I suspect the mmapv1 is not available on those server versions on that platform.